Fabric.Authorization.AccessControl/src/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>Fabric.Authorization.AccessControl</title>
-  <base href="/">
+  <base href="CLIENT_ROOT">
```

Comment:
When we run ng build or npm run build, will this get replaced? If it does, then it will break the replacement logic in the middleware component.

Reply:
We could just make the token in the middleware `<base href="/">`

Reply:
So this works fine.  Running an "ng build" will not replace it.  When our midware runs, then that will replace it with the base url of the running site.  

Now, if we run "ng build --base-href=..."  This is the case where it will replace that value, however I removed this from our setup scripts.  We will be fine if we leave everything as "ng build" or "npm run build"

Reply:
Has it also been removed from our build process?

Reply:
i did not check that.  I can do so

Reply:
Looks like we just run npm build in the build phase.

In the release phase, there are no other things running, besides the npm packaging for Access Control.

Looks like its good on that front.  I will leave it open until after standup.  Meanwhile I will branch off it and do some unit tests.